.travis.yml

```diff
@@ -1,12 +1,22 @@
+sudo: false
+
 language: node_js
 
 node_js:
   - "0.10"
 
+addons:
+  apt:
+    sources:
+      - git-core
```

Comment:
Interesting.  It looks like that whitelist entry was only added 18 days ago in https://github.com/travis-ci/apt-source-whitelist/pull/225.  This is fine with me then, because I don't feel like PR'ing the addition of `git` to the whitelist :stuck_out_tongue_winking_eye: